docs/tutorials/search-form.md

```diff
@@ -20,7 +20,7 @@ In this tutorial, we are going to implement the following features:
 | name | `string` | "" | Display title of the form |
 | order | `string` |  | Visualization order in the dropdown  |
 | default | `boolean` |  | if the search has to be used as default search  |
-| aca:fields| `string[]`| | list of aspects property to add in the query and search in the value for the given text. The property will be concatenated in AND|
+| app:fields| `string[]`| | list of aspects property to add in the query and search in the value for the given text. The property will be concatenated in AND|
```

Comment:
ACA part looks okay but as I mentioned previously this setting should be also added and handled in ADF in the same ticket imo otherwise customers building apps on ADF components won't be able to access this

Reply:
to be continued on Slack